ngsi_adapter/src/test/acceptance/settings/dev-properties.json

```diff
@@ -1,23 +1,31 @@
 {
     "environment": {
-        "name": "experimentation"
+        "name": "experimentation",
+        "log_path": "./logs",
+        "local_path_remote_logs": "./logs/remote/",
+        "default_parser": "qa_probe",
+        "default_parser_data": "qa_probe_valid_template",
+        "default_parser_parameters": [{"key":"NUM_USERS_VALUE", "value":"5"}]
     },
     "monitoring_adapter_service": {
         "protocol": "http",
         "host": "130.206.81.245",
         "port": "1337",
         "resource": "",
-        "host_user": "",
-        "host_password": ""
```

Comment:
They are de user/password of the HOST where the service you are trying to test is deployed.
In this case, that configuration is not used  yet, because we are using private key auth.